runtime/lua/vim/lsp.lua

```diff
@@ -1316,8 +1316,8 @@ function lsp.buf_request_all(bufnr, method, params, callback)
     end
   end)
 
-  local function _sync_handler(err, _, result, client_id)
-    request_results[client_id] = { error = err, result = result }
+  local function _sync_handler(err, result, context)
```

Comment:
naming nit: any objection to using `ctx` for this arg everywhere? it's going to be all over the place I guess, and would rather establish that convention now. `ctx` is a common convention for "context" and has the mild benefit of disambiguating unrelated uses of the word "context" e.g. with `grep ctx`.

Reply:
I'm fine with it, wdyt @neovim/lsp (@tjdevries)?

Reply:
Sounds good to me :+1: